scaleway/table_scaleway_account_project.go

```diff
@@ -0,0 +1,133 @@
+package scaleway
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/account/v3"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableScalewayAccountProject(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "scaleway_account_project",
+		Description: "A Scaleway Account Project.",
+		List: &plugin.ListConfig{
+			Hydrate: listAccountProjects,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "project_id",
```

Comment:
```suggestion
				Name:        "id",
```

Since we're already in the `scaleway_account_project` table, I think it's ok if we simplify the `id` column to use the API field name